components/ProfileMenu.vue

```diff
@@ -5,16 +5,24 @@ import { Menu, MenuButton, MenuItem, MenuItems, Switch } from "@headlessui/vue";
 const profileStore = useProfileStore();
 const { t } = useI18n();
 
-const colorMode = useColorMode();
-const colorTheme = computed(() => {
-  switch (colorMode.preference) {
+const showInterfaceLanguageDialog = ref(false);
+const showContentLanguageDialog = ref(false);
+const showThemeDialog = ref(true);
```

Comment:
Should this really be set to `true` ..? Won't this show the dialog when rendering this component?

Reply:
Ooops. I used it for easier debugging.